charts/signoz/values.yaml

```diff
@@ -554,11 +554,10 @@ frontend:
     #    hosts:
     #      - frontend.domain.com
 
-  # -- Nginx congifmap customize for new parameters
-  customConfig: false
-  # customConfig: true
-  # data: |
-  #   large_client_header_buffers 4 256k;
+  # -- Frontend Nginx extra configurations
+  nginxExtraConfig: |
```

Comment:
added `|` in order to add all extra configs as new line.

Reply:
Yeah, looks good.